addon/components/table-columns.js

```diff
@@ -32,6 +32,12 @@ export default Ember.Component.extend({
   */
   rowClasses: null,
 
+  /**
+    Name of data property for row groups
+    @public
+  */
+  rowGroupDataName: Ember.computed.readOnly('table.rowGroupDataName'),
```

Comment:
lets change like 10 to `computed: { readOnly }` and make this line`rowGroupDataName: readOnly('table.rowGroupDataName')`